File: app/reducers/views/notification.js
// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
// See License.txt for license information.

import {ViewTypes} from 'app/constants';

export default function notification(state = null, action) {
    switch (action.type) {
    case ViewTypes.NOTIFICATION_CHANGED:
        return action.data;
    default:
        return state;
    }
}

File: ios/MattermostShare/Extensions/Notification.swift
//
//  Notification.swift
//  MattermostShare
//
// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
// See LICENSE.txt for license information.
//

import Foundation
import SwiftUI

extension Notification {
  var keyboardHeight: CGFloat {
    return (userInfo?[UIResponder.keyboardFrameEndUserInfoKey] as? CGRect)?.height ?? 0
  }
}
